Protocol.md

```diff
@@ -56,15 +56,26 @@ If both parties already have this information, transmission is unnecessary.
 #### A's fund amount (satoshi)
 How much A inputs into the contract.
 
-#### A's extended public key
-The extended public key to be used to [derive](https://github.com/discreetlogcontracts/dlcspecs/blob/master/KeyDerivation.md) public keys for the Fund transaction, refund transaction and CETs.
+#### A's public keys
+The following public keys:
+
+1. Funding Public Key
+2. CET ToLocal Public Key
+
+#### A's addresses
+
+The following addresses:
+
+1. Funding Change Address
+   - The address to use to send the change if the sum of A's inputs is greater than the fund amount plus the fees.
+2. ToRemote Address
+   - The address to which remote funds will be sent in the unilateral case.
```

Comment:
I would say `in the case of a unilateral close of the contract` to be a bit more precise.

Reply:
Or maybe mention that it is to be used in the `CETs`.